src/app/components/results/Result.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from "react";
+import { Book, Drawing, Guess } from "../../../types/models";
+import { makeStyles, Container, Typography } from "@material-ui/core";
+import CanvasDraw from "react-canvas-draw";
+import uniqid from 'uniqid';
+import PageWrapper from "../PageWrapper";
+
+const useStyles = makeStyles((theme) => ({
+    container: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        paddingTop: '2em',
+    },
+    resultItem: {
+        border: '2px grey groove',
+    },
+    lastWord: {
+        border: '2px darkcyan groove',
+        backgroundColor: 'cadetblue',
+        textColor: 'white',
+    }
+
+}));
+
+interface Props {
+    book: Book;
+
+}
+const Result: FC<Props> = ({ book }) => {
+    const { owner, startWord, pages } = book;
+    const classes = useStyles();
+
+    const isDrawing = (page: Drawing | Guess): page is Drawing => {
```

Comment:
I models.ts, kan vi definere `type Page = Drawing | Guess`?
Også tror det ville vært ryddig om vi hadde lagd en egen fil for type-guards, så det er litt lettere å skjønne at det er det det er snakk om. f.eks`src/types/type-guards.ts`